pkg/webui/console/views/device-general-settings/network-server-form/index.js

```diff
@@ -111,12 +111,19 @@ const NetworkServerForm = React.memo(props => {
         'class_b',
         'class_c',
         'mac_settings',
+        'f_nwk_s_int_key',
+        's_nwk_s_int_key',
+        'nwk_s_enc_key',
       ])
 
       setError('')
       try {
         // Always submit current `mac_settings` values to avoid overwriting nested entries.
-        await onSubmit({ ...updatedValues, mac_settings: castedValues.mac_settings })
+        await onSubmit({
+          ...updatedValues,
+          mac_settings: castedValues.mac_settings,
+          session: { ...updatedValues.session, keys: castedValues.session.keys },
+        })
```

Comment:
It's currently not possible to remove an existing key (by removing the value and submitting it as emtpy), since `diff()` will not include such keys.